code/game/objects/items/hfblade.dm

```diff
@@ -0,0 +1,142 @@
+// Chaplains begone!
+/obj/item/nullrod/scythe/vibro
+	name = "high fronquency bladd"
+
+/obj/item/melee/hfblade
+	name = "high frequency blade"
+	desc = "RULES OF NATURE"
+	icon_state = "hfblade"
+	item_state = "hfblade"
+	icon = 'icons/obj/hfblade.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	force = 0.001
+	throwforce = 20
+	w_class = WEIGHT_CLASS_BULKY
+	block_chance = 50
+	armour_penetration = 75
+	sharpness = IS_SHARP
+	attack_verb = list("chopped", "sliced", "cut", "zandatsu'd")
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	materials = list(MAT_METAL = 10000)
+	light_color = "#40ceff"
+	var/rules_of_nature = TRUE // Turn this off to break the rules, and watch the horrors of no proximity flags unfold.
+	var/brazil = FALSE
+	var/brightness = 5
+
+/obj/item/melee/hfblade/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 30, 95, 5)
+	set_light(brightness)
+	START_PROCESSING(SSobj, src)
```

Comment:
Not sure why we are processing when we never define process(), do you know @ike709 ?

Reply:
No idea. Why would a glorified sword need to process at all anyways?

Reply:
Please remove all processing as it doesn't appear to be needed